runtime/compiler/control/CompilationRuntime.hpp

```diff
@@ -47,7 +47,10 @@
 
 extern "C" {
 struct J9Method;
-struct J9JITConfig;
+struct J9JITConfig
```

Comment:
This is a forward declaration of the J9JITConfig struct. The definition is in j9jitnonbuilder https://github.com/eclipse-openj9/openj9/blob/master/runtime/oti/j9nonbuilder.h#L3726